otherlibs/unix/channels_win32.c

```diff
@@ -21,6 +21,7 @@
 #include <caml/memory.h>
 #include <caml/platform.h>
 #include "caml/unixsupport.h"
+#include <stdbool.h>
```

Comment:
I'm not sure why you've put `#include <stdbool.h>` in these files rather than just having it `platform.h`?

Reply:
I had a memory of something like that (related to https://github.com/ocaml/ocaml/pull/13988/commits/619036af5a1280a347441f586ac8de617b83c977) but in fact since #12964 `domain.h` pulls in `<stdbool.h>`, which is pulled in by `memory.h` when `CAML_INTERNALS` is defined and all of these files here do that.